client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -94,15 +153,23 @@ class GSuiteAddUsers extends React.Component {
 					return <QueryEmailForwards key={ domain.domain } domainName={ domain.domain } />;
 				} ) }
 				<SectionHeader label={ translate( 'Add G Suite' ) } />
-				{ gsuiteUsers ? (
-					<AddEmailAddressesCard
-						domains={ domains }
-						isRequestingSiteDomains={ isRequestingDomains }
-						gsuiteUsers={ gsuiteUsers }
-						planType={ planType }
-						selectedDomainName={ selectedDomainName }
-						selectedSite={ selectedSite }
-					/>
+				{ gsuiteUsers && gSuiteSupportedDomains && ! isRequestingDomains ? (
+					<Card>
+						<GSuiteNewUserList
+							extraValidation={ user => validateAgainstExistingUsers( user, gsuiteUsers ) }
+							domains={ gSuiteSupportedDomains }
+							onUsersChange={ this.onUsersChange }
+							selectedDomainName={ getEligibleGSuiteDomain( selectedDomainName, domains ) }
+							users={ users }
+						>
+							<div className="gsuite-add-users__buttons">
+								<Button>{ translate( 'Cancel' ) }</Button>
```

Comment:
The `Cancel` button doesn't do anything now.

Reply:
Whoops!